api/src/test/java/ai/djl/util/ZipUtilsTest.java

```diff
@@ -49,13 +51,20 @@ public void testEmptyZipFile() throws IOException {
     public void testOffendingTar() throws IOException {
         Path path = Paths.get("src/test/resources/offending.tar");
         Path output = Paths.get("build/output");
-        Path file = output.resolve("tmp/empty.txt");
-        Utils.deleteQuietly(file);
         Files.createDirectories(output);
         try (InputStream is = Files.newInputStream(path)) {
-            TarUtils.untar(is, output, false);
+            Assert.assertThrows(() -> TarUtils.untar(is, output, false));
+        }
+    }
+
+    @Test
+    public void testLinuxCreatedWindowsUsedOffendingTar() throws IOException {
+        TestRequirements.windows();
+        Path tarPath = Paths.get("src/test/resources/linux_create_windows_use.tar");
+        Path output = Paths.get("C:/out");
```

Comment:
Why choose "C:/out"?, better output to `build` folder, although the test throws exception before access to file system but using "c:/out" is a bit confusion:
1. not all machine has c driver
2. not all environment has write access to /out directory